src/Grok.Net/Grok.cs

```diff
@@ -15,6 +15,7 @@ public class Grok
         private readonly Dictionary<string, string> _typeMaps;
         private Regex _compiledRegex;
         private IReadOnlyList<string> _patternGroupNames;
+        private readonly RegexOptions _regexOptions = RegexOptions.Compiled | RegexOptions.ExplicitCapture;
```

Comment:
I'd suggest renaming it to `_defaultRegexOptions`. Do you agree?

Reply:
L147 will look confusing if we change it. I could split this into two variables - `const RegexOptions DefaultRegexOptions` and `_regexOptions` that will be set in constructor.

Reply:
Yes please, let's try this way